python/pyspark/ml/tests.py

```diff
@@ -929,6 +932,50 @@ def test_apply_binary_term_freqs(self):
                                    ": expected " + str(expected[i]) + ", got " + str(features[i]))
 
 
+class ALSTest(PySparkTestCase):
+
+    def test_storage_levels(self):
```

Comment:
+1 on removing this. We just need to test the default values and the setters actually set the value on JVM side.